packaging/docker/build.sh

```diff
@@ -29,7 +29,7 @@ if [ "${VERSION}" == "" ]; then
 fi
 
 # If we are not in netdata git repo, at the top level directory, fail
-GIT_TOP_LEVEL=$(basename "$(git rev-parse --show-toplevel)")
+TOP_LEVEL=$(basename "$(git rev-parse --show-toplevel)")
```

Comment:
why not just do
```
cd $(git rev-parse --show-toplevel)
```
and remove whole checking section?

Reply:
You guarantee that the script will always run within a git repo?

Reply:
Why wouldn't it?

Reply:
I am thinking about the customer perspective, not about how we intent to use this.
If a customer decides to mistakenly use this outside of a git repo, executing the cd may fail.
It is safer to just parse and inform the user about the failure.

Reply:
Customer is not supposed to build images from this.